### PR TITLE
data: SGsubhash77

### PR DESCRIPTION
**Pull request by SGsubhash77**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5599"></a>

### CODE REVIEW

File: data/SGsubhash77 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+     {
+       "name": "Follow me on LinkedIn",
+       "url": "https://www.linkedin.com/in/subhash-gandhi/",
+       "icon": "FaLinkedin"
+     },
+      {
+       "name": "Follow me on Twitter",
+       "url": "https://www.twitter.com/in/subhashgandhi89087282/",
+       "icon": "FaTwitter"
+     },
```
Comment (khushalarora) [inline data/SGsubhash77:25]: Remove this **Comma** after the **' closing brace  }'**  :+1: 
```Suggestion
     }
```

File: data/SGsubhash77 [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+   ],
```
Comment (khushalarora) [inline data/SGsubhash77:26]: Remove this **' Comma '** also  :+1:

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @SGsubhash77, thank you for contributing  :tada: 
Please rename your file and add  **.json**  at the end.
It will look like  `data/SGsubhash77.json`

File: data/SGsubhash77 [hunk 3]
```diff
@@ -0,0 +1,28 @@
+ {
+   "name": "Subhash Gandhi",
+   "type": "personal",
+   "bio": "**Open Source**  Enthusiast | Full Stack Dev",
+   "tags": ["Open Source", "JavaScript", "Full Stack", "MERN Stack", "Node.js" ,"Ui/Ux"],
+   "socials": [
+     { "icon": "FaGithub", "url": "https://github.com/SGsubhash77.png"},
+     { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/subhash-gandhi/" },
```
[the remark this answers is not in the document]
Comment (SGsubhash77) [reply data/SGsubhash77]: ok sure, I will resolve it.

**Review by SGsubhash77 — COMMENTED** [review]
I have made the changes you have told @khushalarora

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by SGsubhash77** [comment]
Thankyou for your comment making my first contribution

**Pull request merged**